ts/web3js/README.md

```
@@ -0,0 +1,18 @@
+# @magicblock-labs/ephemeral-rollups-sdk
+
+TypeScript helpers for preparing and routing transactions to Ephemeral Rollups on Solana.
+
+Quickstart and integration guide:
+https://docs.magicblock.gg/pages/get-started/how-integrate-your-program/quickstart
+
+## Install
+
+npm install @magicblock-labs/ephemeral-rollups-sdk
+# or
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)